Formula/libthemis.rb

```diff
@@ -1,34 +1,34 @@
 class Libthemis < Formula
-  desc 'High-level cryptographic primitives'
-  homepage 'https://www.cossacklabs.com/themis'
-  head 'https://github.com/cossacklabs/themis.git'
-  url 'https://github.com/cossacklabs/themis/archive/0.14.0.tar.gz'
-  sha256 '2efb793e0ef604fb97258b07671a83135ad9229d83b92d7758b43510dcc6cb07'
-  version '0.14.0'
+  desc "High-level cryptographic primitives"
+  homepage "https://www.cossacklabs.com/themis"
+  url "https://github.com/cossacklabs/themis/archive/0.15.1.tar.gz"
```

Comment:
we will need to upload gz to github, right?

Reply:
No. It is a GitHub feature. It is already there.